docs/reference/errors-and-warnings/NU1105.md

```diff
@@ -12,10 +12,29 @@ f1_keywords:
 
 # NuGet Error NU1105
 
-<pre>Unable to read project information for 'ProjectFile'. The project file may be invalid or missing targets required for restore.</pre>
+<pre>Unable to read project information for 'ProjectFile'. The project file may be invalid/missing targets required for restore or the project might be unloaded.</pre>

```

Comment:
I believe this is usually on par with the actual error message.
So if we're changing it here, we should change it in the product as well.